### ISSUE TITLE
Problem with Application Specific Remapping

### ISSUE THREAD

**Issue by ziontee113** [rvaiya/keyd#225]
Hi, I'm new to Linux and I'm trying to remap C.j and C.k to Down and Up keys in Firefox.

I installed `keyd` through `paru keyd`. I tried the Quick Start step in `/etc/keyd/default.conf`, which worked. But when I attempt to try Application Specific Remapping, I'm having some troubles.

I did `usermod -aG keyd myusername` & created the `~/.config/keyd/app.conf` file. In there I put:

```
[firefox]

alt.[ = C-S-tab
alt.] = macro(C-tab)
```

When I change windows, I see prompts that detect the Active Window, and also the prompt `bind: Connection refused` or `bind: No such file or directory`

When I try another command, `keyd -m`, it says `failted to open /dev/input/event6` event0 to event17. Then I tried `sudo keyd -m`, it seems that the functionality works fine.

I did try `sudo keyd-application-mapper` but now this time it says `ERROR: could not find app.conf, make sure it is in ~/.config/keyd/app.conf`.

I'm really confused so I opened this issue. I think it's my fault but I really don't know how to resolve this. Sorry for bothering you.

Also if I could make `keyd-appication-mapper` work, how do I remap C-j and C-k to [firefox], thank you :)

- Edit: I tried again and this time `keyd-application-mapper` doesn't tell errors, but the mappings still won't work.

**Comment by ziontee113** [comment]
It's so weird. I tried every key possible and it still won't work. `keyd-application-mapper` clearly detects the active window, but it won't map the keys.
I tried [firefox] and [kitty], multiple different keymaps and still nothing.

I put those in my `~/.config/keyd/app.conf`, am I doing something wrong?

**Comment by ziontee113** [comment]
Nope, still doesn't work.
So I tried:
```
newgrp keyd
sudo usermod -aG keyd myusername
keyd-appication-mapper
```
The behavior is still the same. `keyd-application-mapper` still runs, it still detects if I change windows, but it just won't map the keys.
If I delete my `.config/keyd/app.conf` it will instantly error. So it's not like it doesn't detect my app.conf file, it just won't map the keys for some reason.

**Comment by ziontee113** [comment]
Oh for the love of god stupid me!
So it turns out that I deleted the `/etc/keyd/default.conf` file (because I messed up something earlier and I reverted back to Xmodmap to map my Capslock to Escape key.

So it's totally my bad! Sorry @herrsimon and thank you so much for being so patient with me.

So I just went through the whole installation process again. Make sure that `etc/keyd/default.conf` and `~/.config/keyd/apps.conf` exist. Run the correct commands and it works.

Thank you so much :)
